test/FunctionsV2/TestDurableHttpRequest.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Runtime.Serialization;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.Tests
```

Comment:
If you move this to common, you shouldn't need a duplicate version of this file in `test/FunctionsV1` and 'test/FunctionsV2`.